### DISCUSSION TITLE
物語「ものがたり」　Chapter　Ⅰ　~君が見た夢の物語~　Update

### DISCUSSION THREAD

**Discussion by mirai-mamori** [mirai-mamori/Sakurairo#449] [General]
![versions_cn](https://user-images.githubusercontent.com/61381142/155470887-ebe5374f-aca8-4c81-ac04-9fbf39033c63.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/mirai-mamori/Sakurairo/releases/tag/2.4.0'>物語「ものがたり」　Chapter　Ⅰ　~君が見た夢の物語~　Update</a>.</em>

**Comment by madoka-chann** [comment]
来喽！！！！！
